packages/labs/context/package.json

```diff
@@ -31,18 +31,111 @@
-    "test:watch": "npm run test:dev -- --watch",
-    "checksize": "rollup -c --environment=CHECKSIZE",
-    "regen-package-lock": "rm -rf node_modules package-lock.json; npm install"
+    "build": "wireit",
+    "build:ts": "wireit",
+    "build:ts:types": "wireit",
+    "build:rollup": "wireit",
+    "test": "wireit",
+    "test:dev": "wireit",
+    "test:prod": "wireit",
+    "checksize": "wireit"
+  },
+  "wireit": {
+    "build": {
+      "dependencies": [
+        "build:rollup",
+        "build:ts",
+        "build:ts:types",
+        "../../lit:build",
+        "../../reactive-element:build"
+      ]
+    },
+    "build:ts": {
+      "command": "tsc --build --pretty",
+      "dependencies": [
+        "../../lit:build:ts:types",
+        "../../reactive-element:build:ts:types"
+      ],
+      "clean": "if-file-deleted",
+      "files": [
+        "src/**/*.ts"
+      ],
+      "output": [
+        "development",
+        "tsconfig.tsbuildinfo"
+      ]
+    },
+    "build:ts:types": {
+      "command": "treemirror development . \"**/*.d.ts{,.map}\"",
+      "dependencies": [
+        "../../internal-scripts:build",
+        "build:ts"
+      ],
+      "files": [],
+      "output": [
+        "*.d.ts{,.map}",
+        "lib/**/*.d.ts{,.map}",
+        "test/**/*.d.ts{,.map}"
+      ]
+    },
+    "build:rollup": {
+      "command": "rollup -c",
+      "dependencies": [
+        "build:ts"
+      ],
+      "files": [
+        "rollup.config.js",
+        "../../../rollup-common.js"
+      ],
+      "output": [
+        "index.js{,.map}",
+        "lib/**/*.js{,.map}",
+        "test/**/*.js{,.map}"
+      ]
+    },
+    "checksize": {
+      "command": "rollup -c --environment=CHECKSIZE",
+      "dependencies": [
+        "build:ts"
+      ],
+      "files": [
+        "rollup.config.js",
+        "../../../rollup-common.js"
+      ],
+      "output": []
+    },
+    "test": {
+      "dependencies": [
+        "test:dev",
+        "test:prod"
+      ]
+    },
+    "test:dev": {
+      "command": "MODE=dev node ../../tests/run-web-tests.js \"development/**/*_test.js\" --config ../../tests/web-test-runner.config.js",
+      "dependencies": [
+        "build"
+      ],
+      "#comment": "TODO(aomarks) Factor out a file-only target in tests/ to DRY these config files into a single place",
+      "files": [
+        "../../tests/run-web-tests.js",
+        "../../tests/web-test-runner.config.js",
+        "../../tests/wtr-config.js",
+        "../../tests/rollup-resolve-remap.js"
+      ],
+      "output": []
+    },
+    "test:prod": {
+      "command": "MODE=prod node ../../tests/run-web-tests.js \"development/**/*_test.js\" --config ../../tests/web-test-runner.config.js",
+      "dependencies": [
+        "build"
+      ],
+      "files": [
+        "../../tests/web-test-runner.config.js",
+        "../../tests/wtr-config.js",
+        "../../tests/rollup-resolve-remap.js"
```

Comment:
Nit: Missing dep on `"../../tests/run-web-tests.js",`.

No action required in this PR: Depending on a single file group or a single test dependency as a single target would be awesome here, cause we have a bunch of fiddly test files.

Reply:
Done, and for all other packages I found that were missing it.